### PR TITLE
Update `PlacementRule` --> `Placement` for GRC

### PR DESCRIPTION
**Pull request by dhaiducek**
(Setting to "Draft" until `2.10_stage` is open and it can switch over.)

ref: https://issues.redhat.com/browse/ACM-9571

### CODE REVIEW

**Review by yiraeChristineKim — APPROVED** [review]

**Comment by dockerymick** [comment]
@dhaiducek thanks for creating the PR, I updated the branch to 2.10 and will review it shortly. Thanks!

File: governance/create_policy.adoc [hunk 1]
```diff
@@ -17,7 +17,9 @@ You can create a security policy from the command line interface (CLI) or from t
 
 *Required access:* Cluster administrator
 
-*Important:* You must define a placement and placement binding to apply your policy to a specific cluster. Enter a valid value for the _Cluster selector_ field to define a `Placement` and `PlacementBinding`.
+*Important:*
+* You must define a placement and placement binding to apply your policy to a specific cluster. Enter a valid value for the cluster _Label selector_ field to define a `Placement` and `PlacementBinding`.
+* In order to use `Placement`, a `ManagedClusterSet` must be bound to the `Placement` namespace with a `ManagedClusterSetBinding`. Refer to xref:../cluster_lifecycle/create_clustersetbinding.adoc#creating-managedclustersetbinding[Creating a _ManagedClusterSetBinding_ resource] for additional details.
```
Comment (dockerymick) [inline governance/create_policy.adoc]:
```suggestion
* In order to use `Placement` resource, a `ManagedClusterSet` resource must be bound to the namespace of the `Placement` resource with a `ManagedClusterSetBinding` resource. Refer to xref:../cluster_lifecycle/create_clustersetbinding.adoc#creating-managedclustersetbinding[Creating a _ManagedClusterSetBinding_ resource] for additional details.
```

**Review by dockerymick — COMMENTED** [review]
Everything seems pretty good to me! Thanks Dale! Quick question, is PlaceentRule officially removed for 2.10?

**Comment by dhaiducek** [comment]
> Everything seems pretty good to me! Thanks Dale! Quick question, is PlacementRule officially removed for 2.10?

It's not removed that I'm aware of (that'd be a question for App Lifecycle--I think it might be removed in 2.11?), but we wanted to be prepared well before it was removed.

File: governance/create_policy.adoc [hunk 2]
```diff
@@ -17,7 +17,9 @@ You can create a security policy from the command line interface (CLI) or from t
 
 *Required access:* Cluster administrator
 
-*Important:* You must define a placement and placement binding to apply your policy to a specific cluster. Enter a valid value for the _Cluster selector_ field to define a `Placement` and `PlacementBinding`.
+*Important:*
+* You must define a placement and placement binding to apply your policy to a specific cluster. Enter a valid value for the cluster _Label selector_ field to define a `Placement` and `PlacementBinding`.
+* In order to use a `Placement` resource, a `ManagedClusterSet` resource must be bound to the namespace of the `Placement` resource with a `ManagedClusterSetBinding` resource. Refer to xref:../cluster_lifecycle/create_clustersetbinding.adoc#creating-managedclustersetbinding[Creating a _ManagedClusterSetBinding_ resource] for additional details.
```
Comment (dockerymick) [inline governance/create_policy.adoc]:
```suggestion
* In order to use a `Placement` resource, a `ManagedClusterSet` resource must be bound to the namespace of the `Placement` resource with a `ManagedClusterSetBinding` resource. Refer to link:../cluster_lifecycle/create_clustersetbinding.adoc#creating-managedclustersetbinding[Creating a _ManagedClusterSetBinding_ resource] for additional details.
```

File: governance/create_policy.adoc [hunk 3]
```diff
@@ -17,13 +17,9 @@ You can create a security policy from the command line interface (CLI) or from t
...
-
-See link:https://kubernetes.io/docs/concepts/overview/working-with-objects/labels/#resources-that-support-set-based-requirements[Resources that support support set-based requirements] in the Kubernetes documentation for a valid expression. View the definitions of the objects that are required for your {product-title-short} policy:
-
-* _PlacementBinding_: Binds the placement.
-
-View more descriptions of the policy YAML files in the xref:../governance/policy_overview.adoc#policy-overview[Policy overview]. 
+*Important:*
+* You must define a placement and placement binding to apply your policy to a specific cluster. The `PlacementBinding` resource binds the placement. Enter a valid value for the cluster _Label selector_ field to define a `Placement` and `PlacementBinding` resource.
+* In order to use a `Placement` resource, a `ManagedClusterSet` resource must be bound to the namespace of the `Placement` resource with a `ManagedClusterSetBinding` resource. Refer to xref:../clusters/cluster_lifecycle/create_clustersetbinding.adoc#creating-managedclustersetbinding[Creating a _ManagedClusterSetBinding_ resource] for additional details.
```
Comment (dockerymick) [inline governance/create_policy.adoc]:
```suggestion
* In order to use a `Placement` resource, a `ManagedClusterSet` resource must be bound to the namespace of the `Placement` resource with a `ManagedClusterSetBinding` resource. Refer to link:../clusters/cluster_lifecycle/create_clustersetbinding.adoc#creating-managedclustersetbinding[Creating a _ManagedClusterSetBinding_ resource] for additional details.  
```

File: governance/policy_set_ctrl.adoc [hunk 4]
```diff
@@ -7,7 +7,9 @@ Additionally, when a policy is a part of multiple policy sets, existing and new
 
 *Deprecated:* `PlacementRule`
 
-*Note:* The {product-title-short} hardening sample policy set uses cluster placement. If you use cluster placement, bind the namespace containing the policy to the managed cluster set. See link:../gitops/deploy_gitops.adoc#deploying-policies-to-your-cluster[Deploying policies to your cluster] for more details on using cluster placement.
+*Note:*
+* The {product-title-short} hardening sample policy set uses cluster placement. If you use cluster placement, bind the namespace containing the policy to the managed cluster set. See link:../gitops/deploy_gitops.adoc#deploying-policies-to-your-cluster[Deploying policies to your cluster] for more details on using cluster placement.
```
Comment (dockerymick) [inline governance/policy_set_ctrl.adoc]:
```suggestion
* The {product-title-short} sample policy set uses cluster placement. If you use cluster placement, bind the namespace containing the policy to the managed cluster set. See link:../gitops/deploy_gitops.adoc#deploying-policies-to-your-cluster[Deploying policies to your cluster] for more details on using cluster placement.
```

File: governance/policy_set_ctrl.adoc [hunk 5]
```diff
@@ -7,7 +7,10 @@ Additionally, when a policy is a part of multiple policy sets, existing and new
 
 *Deprecated:* `PlacementRule`
```
Comment (dockerymick) [inline governance/policy_set_ctrl.adoc]: I noticed that you removed this in other files, I assume that we also want to remove this instance as well?

**Review by jc-berger — APPROVED** [review]
/lgtm

File: governance/policy_set_ctrl.adoc (hunk 5 above)
Comment (dhaiducek) [reply governance/policy_set_ctrl.adoc]: Good catch, yes we do!

**Comment by dhaiducek** [comment]
Thanks for the reviews! It looked like the `../clusters/cluster_lifecycle/` paths are now `../cluster_lifecycle`, so I updated the rest of them to the new path. Also, I see you were updating `xref` to `link`, so I updated some others contained in this PR.

**Comment by dockerymick** [comment]
@jc-berger thanks for your review and @dhaiducek thanks for accepting my suggestions and changes. Approving and merging this PR

**Review by dockerymick — APPROVED** [review]

**Pull request merged**